### PR TITLE
Translate 2017 Fukuoka Ruby Award Competition (pt)

### PR DESCRIPTION
**Pull request by jcserracampos**

### CODE REVIEW

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+programa Ruby interessante, por favor sinta-se encorajado a participar.
+
+2017 Fukuoka Ruby Award Competition - Grande Prêmio - 1 Milhão de Yens!
+
+Prazo de inscrição: 27 de dezembro de 2016
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de painelista selecionarão os vencedores da Fukuoka Competition.
+O grande prêmio para a Fukuoka Competition são um milhão de yens.
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: `é um milhão de yens` (ou ienes)

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Ganhadores do grande prêmio anteriores incluem Rhomobile (EUA) e APEC Climate Center(Coreia)
+
+[http://myfukuoka.com/category/news/ruby-news/](http://myfukuoka.com/category/news/ruby-news/)
+
+Programas inscritos na competição não precisam ser escritos inteiramente em Ruby
+mas devem aproveitar-se de uma das características únicas do Ruby.
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: This sentence is correct, I just think it sounds better to write it without `de uma`, like `aproveitar-se das características...`.

With `de uma` it sounds like you're waiting for the unique characteristic. 
Anyway, just pointing that out.

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* Amazon Fire Tablet (sujeito a alteração)
+* Consultoria técnica de arquiteto da AWS
+
+O vencedor do GMO Pepavo Prize receberá:
+
+* Uma cesta de presentes cheia de comidas locais e petiscos (no valor de 30.000 yens)
+* Um certificado de 50.000 yens para serviços de domínio
+
+O vencedor do IIJ GIO Prize receberá:
+* Um cupom gratuíto no valor de 500.000 yens do IIJ GIO (até 6 meses)
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: `gratuito`

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+O vencedor do Salesforce Prize receberá:
+
+* Novidades da salesforce.com
+
+"Matz estará testando e revisando completamente seu código fonte, então
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: `irá testar e revisar completamente..` I think it sounds better without the "gerundismo"

**Comment by jcserracampos** [comment]
Dunno why I closed this.

**Comment by jcserracampos** [comment]
@fpgentil Can you check this?

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+convidar para se inscrever na seguinte competição Ruby. Se você já desenvolveu um
+programa Ruby interessante, por favor sinta-se encorajado a participar.
+
+2017 Fukuoka Ruby Award Competition - Grande Prêmio - 1 Milhão de ienes!
+
+Prazo de inscrição: 27 de dezembro de 2016
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de painelista selecionarão os vencedores da Fukuoka Competition.
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: `um grupo de painelistas` ?

Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: I'd leave _Fukuoka Competition_ in italic.

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+O grande prêmio para a Fukuoka Competition são um milhão de ienes.
+Ganhadores do grande prêmio anteriores incluem Rhomobile (EUA) e APEC Climate Center(Coreia)
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: `Ganhadores dos grandes prêmios anteriores` .... it's missing an empty space between `Center (Coreia)`

File: pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+programa Ruby interessante, por favor sinta-se encorajado a participar.
+
+2017 Fukuoka Ruby Award Competition - Grande Prêmio - 1 Milhão de ienes!
+
+Prazo de inscrição: 27 de dezembro de 2016
+
+![Fukuoka Ruby Award](http://www.digitalfukuoka.jp/javascripts/kcfinder/upload/images/fukuokarubyaward2017.png)
+
+Matz e um grupo de painelista selecionarão os vencedores da Fukuoka Competition.
+O grande prêmio para a Fukuoka Competition são um milhão de ienes.
```
Comment (fpgentil) [inline pt/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: Shouldn't it be `é um milhão de ienes`? 95% sure here!

**Comment by jcserracampos** [comment]
@fpgentil I fixed and amended.

**Pull request merged**